transport/transport_test.go

```diff
@@ -51,7 +51,6 @@ func removeIfPresent(path string) error {
 }
 
 func TestMain(m *testing.M) {
-	flag.IntVar(&log.Level, "loglevel", log.LevelInfo, "log level (0 = DEBUG, 4 = ERROR)")
 	flag.Parse()
```

Comment:
We should remove `flag` from this file entirely. (It's not longer parsing anything)